setup.cfg

```diff
@@ -14,4 +14,6 @@ include_trailing_comma=true
 known_first_party=dvc,tests
 known_third_party=PyInstaller,RangeHTTPServer,boto3,colorama,configobj,distro,dpath,flaky,flufl,funcy,git,google,grandalf,mock,mockssh,moto,nanotime,networkx,packaging,paramiko,pathspec,pytest,requests,ruamel,setuptools,shortuuid,tqdm,voluptuous,yaml,zc
 line_length=79
+force_grid_wrap=0
+use_parentheses=True
```

Comment:
https://black.readthedocs.io/en/stable/the_black_code_style.html#how-black-wraps-lines

Reply:
Why I added this is because, isort adds a `\` on hanging indent for imports:

```py
from dvc.stage.serialize import \
    to_single_stage_lockfile as _to_single_stage_lockfile
```

Whereas black formats it into this:
vs:
```py
from dvc.stage.serialize import (
    to_single_stage_lockfile as _to_single_stage_lockfile
)
```